core/server/data/migrations/versions/3.12/01-remove-legacy-is-paid-flag-from-settings.js

```diff
@@ -0,0 +1,110 @@
+const _ = require('lodash');
+const common = require('../../../../lib/common');
+const debug = require('ghost-ignition').debug('migrations');
+
+module.exports.config = {
+    transaction: true
+};
+
+module.exports.up = (options) => {
+    let localOptions = _.merge({
+        context: {internal: true}
+    }, options);
+    const settingsKey = 'members_subscription_settings';
+
+    return localOptions
+        .transacting('settings')
```

Comment:
Could this not be more specific like:
```js
.transacting('settings')
.where('key', settingsKey)
.select('value')
.first()
```

Reply:
True! I like it. Will redo the query :+1: